REBASE.openshift.md

```diff
@@ -408,3 +408,90 @@ git log v1.21.1..v1.21.2 --ancestry-path --reverse --no-merges
 6. Update openshift dependencies and re-run `hack/update-vendor.sh`.
 7. Update kubernetes version in `openshift-hack/images/hyperkube/Dockerfile.rhel`.
 8. Run `make update` see [Updating generated files](#updating-generated-files).
+
+##  Z stream release
+
+1. Clone kubernetes repository:
+
+```
+git clone https://github.com/kubernetes/kubernetes
+```
+
+2. Create openshift remote
+
+```
+git remote add openshift `https://github.com/openshift/kubernetes`
+```
+
+3. Fetch everything from the remote `openshift`
+
+```
+git fetch openshift
+```
+
+4. Checkout to release-4.8 branch
```

Comment:
How about we put something like release-Y.X branch  (as an example  for 4.8 it would be release-4.8)

Reply:
Addressed this comment.